tests/facebook.rb

```diff
@@ -5,8 +5,8 @@
 require 'uri'
 
 status = 0
-diff = `git diff origin/master...HEAD entries/ | grep "^+[[:space:]]*\\"facebook\\":" | cut -c21-`
-diff.gsub("\n", '').gsub(',', '').split('"').each do |page|
+diff = `git diff origin/master...HEAD entries/ | sed -n 's/^+.*"facebook"[^"]*"\\(.*\\)".*/\\1/p'`
+diff.split('\n').each do |page|
```

Comment:
```suggestion
diff.split(' ').each do |page|
```
I might be missing something here, but when I run this, I get the error:
```
bad URI(is not URI?): "https://www.facebook.com/pg/myinvalidfacebookpage\\n" (URI::InvalidURIError)
```
There seems to be a `\n` that isn't getting removed. The test works when splitting at new lines is changed to splitting at spaces or removing the `\n` from the string.

Reply:
Nice catch.

I have no clue why that would be the case. The variable `diff` is literally `rbc\nvygruppen\n`. No spaces, just linebreakers. Splitting with `' '` does indeed fix it though.
Maybe @phallobst knows what is happening?